device-types/Nokia/7220-IXR-H4-64QSFPDD-2SFPP.yaml

```diff
@@ -0,0 +1,102 @@
+---
+manufacturer: Nokia
+model: 7220 IXR-H4 64QSFPDD 2SFP+
+slug: nokia-7220-ixr-h4-64qsfpdd-2sfpp
+part_number: 3HE17750AA
+u_height: 2
+is_full_depth: true  # 25.56"
+weight: 47.39
+weight_unit: lb
```

Comment:
Do you have the `airflow` data point for this device or is it changeable based on the fan module?

Reply:
H4 only supports front-to-rear, so we could 'hardcode' it here

Reply:
updated